src/domain/pricing/editModal/fields/HarborServicesFields.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { FunctionComponent } from 'react';
```

Comment:
Satamapalvelut and Lisäpalvelut seems to have an small alignment issue
<img width="449" alt="Screenshot 2020-05-07 at 9 57 07" src="https://user-images.githubusercontent.com/1407958/81264022-67ff0500-9049-11ea-9b11-4928b0e8ab53.png">

Reply:
Adjusted Select styles to match HDS TextInput